### PR TITLE
Moirai One: Version 1.000; ttfautohint (v1.8.4.7-5d5b);gftools[0.9.28] added

### PR DESCRIPTION
**Pull request by aaronbell**
eb8387f: [gftools-packager] Moirai One: Version 1.000; ttfautohint (v1.8.4.7-5d5b);gftools[0.9.28] added

* Moirai One Version 1.000; ttfautohint (v1.8.4.7-5d5b);gftools[0.9.28] taken from the upstream repo https://github.com/JAMO-TYPEFACE/Moirai at commit https://github.com/JAMO-TYPEFACE/Moirai/commit/3220f0c211c8d3a3bb989896cf7ea66d16a925e6.

### CODE REVIEW

File: ofl/moiraione/METADATA.pb [hunk 1]
```diff
@@ -0,0 +1,23 @@
+name: "Moirai One"
+designer: "Jiyeon Park"
```
[the remark this answers is not in the document]
Comment (aaronbell) [reply ofl/moiraione/METADATA.pb:2]: My understanding of the way the JAMO stuff has been set up is that JAMO is primarily the publisher, with a set of mentors who are instructing / reviewing work done by the students. So ultimately the students are the designer, and JAMO is sort of more of just a publisher. 

How do you think it best to document this?

[the remark this answers is not in the document]
Comment (aaronbell) [reply ofl/moiraione/METADATA.pb:2]: Sounds good! 

So in this case I’d list the designer as  `"Jiyeon Park, JAMO"`?

And we’d remove the JAMO info from the descriptions?

[the remark this answers is not in the document]
Comment (RosaWagner) [reply ofl/moiraione/METADATA.pb:2]: Yep

**Comment by aaronbell** [comment]
I've corrected the description file and added "JAMO" as author.

**Pull request merged**